resources/lang/en/field.php

```diff
@@ -401,4 +401,10 @@
     'undefined_page' => 'Undefined Page',
     'id' => 'ID',
     'custom_field' => 'Custom Field',
+
+    // ads_bar_widget
+    'pending_ads'       => 'Pending Ads',
```

Comment:
this fields already exist

Reply:
@MostafaMoradii

Reply:
under status you mean? @muammertop

Reply:
I checked wasn't a status translation in the adv.

Reply:
yes I checked also